STAR.py

```diff
@@ -6,10 +6,9 @@
 class InvalidElection(Exception):
     pass
 
-class TrueTie:
-    def __init__(self, tied):
-        self.tied = tied
-        self.elected = [] 
+class TrueTie(set):
+    def __repr__(self) -> str:
+        return 'TIE: ' + str(set(self))
```

Comment:
```suggestion
        return 'TrueTie(' + repr(set(self)) + ')'
```

"If at all possible, this should look like a valid Python expression that could be used to recreate an object with the same value"